tests/Checker/Checks/I18n_Usage_Check_Tests.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * Tests for the I18n_Usage_Check_Tests class.
+ *
+ * @package plugin-check
+ */
+
+use WordPress\Plugin_Check\Checker\Checks\I18n_Usage_Check;
+use WordPress\Plugin_Check\Checker\Check_Context;
+use WordPress\Plugin_Check\Checker\Check_Result;
+
+class I18n_Usage_Check_Tests extends WP_UnitTestCase {
+
+	protected $i18n_usage_check;
+
+	public function set_up() {
+		parent::set_up();
+
+		$this->i18n_usage_check = new I18n_Usage_Check();
+	}
```

Comment:
Maybe a bit of a personal preference, but I don't think we need a `set_up` method for something as simple as this. Makes the test code a bit harder to follow, and there isn't really a benefit here avoiding this "duplicate code".

Reply:
Removed it.